Formula/grakn.rb

```diff
@@ -1,20 +1,20 @@
 class Grakn < Formula
   desc "The Database for AI"
   homepage "https://grakn.ai"
-  url "https://github.com/graknlabs/grakn/releases/download/v0.16.0/grakn-dist-0.16.0.tar.gz"
-  sha256 "1c7194f09cbf31949323b79b2fb8a5683381af5ecf37886fd45deb71585261ed"
+  url "https://github.com/graknlabs/grakn/releases/download/v0.17.1/grakn-dist-0.17.1.tar.gz"
+  sha256 "a4d190d95515b01ce736056e2d738d67c14298465ba7fb92df5295612bb91475"
 
   bottle :unneeded
 
   depends_on :java => "1.8+"
 
   def install
     libexec.install Dir["*"]
-    bin.install Dir["#{libexec}/bin/*"]
-    bin.env_script_all_files(libexec/"bin", :CASSANDRA_HOME => ENV["CASSANDRA_HOME"])
+    bin.install Dir["#{libexec}/grakn", "#{libexec}/graql"]
+    bin.env_script_all_files(libexec/"", :SERVICE_HOME => ENV["SERVICE_HOME"])
```

Comment:
`SERVICE_HOME` needs to have a default value set in the formula or we should just be using
```
libexec.install Dir["*"]
bin.write_exec_script libexec/"grakn", libexec/"graql"
```